src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -6,12 +6,34 @@
 
 namespace Neo.SmartContract.Native
 {
+    /// <summary>
+    /// A native contract that manages the system policies.
+    /// </summary>
     public sealed class PolicyContract : NativeContract
     {
+        /// <summary>
+        /// The default execution fee factor.
+        /// </summary>
         public const uint DefaultExecFeeFactor = 30;
+
+        /// <summary>
+        /// The default storage price.
+        /// </summary>
         public const uint DefaultStoragePrice = 100000;
+
+        /// <summary>
+        /// The default network fee per byte of transactions.
+        /// </summary>
         public const uint DefaultFeePerByte = 1000;
+
+        /// <summary>
+        /// The maximum execution fee factor that the committee could set.
```

Comment:
maybe "can" instead of "could", the "could" gave me a little bit of impression from the past, but I believe it is also correct as it is.

Reply:
Updated.